### PR TITLE
rework ocm related documentation

### PR DESCRIPTION
**Pull request by fabianburth**
**What this PR does / why we need it**:
This PR reworks the OCM related sections of the Guided Tour to make them more accessible. 

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```doc user
NONE
```

### CODE REVIEW

File: docs/guided-tour/blueprints/helm-chart-resource/README.md [hunk 1]
```diff
@@ -1,81 +1,114 @@
-# Helm Chart Resources in the Component Descriptor
+# Helm Chart Resources in the Component
```
Comment (achimweigel) [inline docs/guided-tour/blueprints/helm-chart-resource/README.md:1]: What about referencing a helm chart in a helm chart repository? This is currently our most important customer use case.

**Review by achimweigel — APPROVED** [review]
only two small remarks

File: docs/guided-tour/blueprints/external-blueprint/README.md [hunk 2]
```diff
@@ -23,34 +23,55 @@ registry, from where the Landscaper can access it. You can find the commands, wh
...
-An Installation may reference its blueprints via so-called
-[component-descriptors](../../../concepts/Glossary.md#_component-descriptor_).  A component descriptor describes a
-component, or rather, a specific component version. In general, a component version is a container for all required
-resources for the deployment of a specific version of an application or software system. In this example, the
-application is the hello-world application deployable with the landscaper through an external blueprint. Thereby, the
-external blueprint is the only resource required for the deployment and thus, the only resource contained in the
-component version. A component version may either contain a resource through referencing it at an external location 
-(such as an OCI registry) or through embedding it as a local blob. A quick introduction to this concepts can be found at
-the end of this section. 
+To consume the blueprint from such a referencable location, it has to be contained in a component.
```
Comment (achimweigel) [inline docs/guided-tour/blueprints/external-blueprint/README.md:32]: I found these explanations very good and helpful. You introduced them some time before and I was very happy about them.

File: docs/guided-tour/blueprints/external-blueprint/README.md [hunk 3]
```diff
@@ -95,46 +116,6 @@ kubectl apply -f <path to target.yaml>
...
-Again, the commands used to create and upload the above component versions can be found
-[here](./commands/upload-component-version.sh).  
-If you want to inspect the uploaded component version (e.g. to find out 
-whether we really have uploaded the one with the external resource), you can do so using the following command:
-
-```shell 
-ocm download componentversion \
-  eu.gcr.io/gardener-project/landscaper/examples//github.com/gardener/landscaper-examples/guided-tour/external-blueprint:2.0.0 \
-  -O component-archive 
-```
```
Comment (achimweigel) [inline docs/guided-tour/blueprints/external-blueprint/README.md:136]: Also this information introduced by you some time ago was quite good. Just my opinion :-(

File: docs/guided-tour/blueprints/external-blueprint/README.md [hunk 4]
```diff
@@ -23,34 +23,55 @@ registry, from where the Landscaper can access it. You can find the commands, wh
 in this script: [commands/push-blueprint.sh](./commands/push-blueprint.sh).
```
[the remark this answers is not in the document]
Comment (fabianburth) [reply docs/guided-tour/blueprints/external-blueprint/README.md]: Done

File: docs/guided-tour/blueprints/helm-chart-resource/README.md (hunk 1 above)
Comment (fabianburth) [reply docs/guided-tour/blueprints/helm-chart-resource/README.md:1]: Added an example of a corresponding component configuration file

**Comment by achimweigel** [comment]
That is a valid argument but the explanations are so helpful that me as a beginner would be very happy to have them. The explanations are even so good that I first thought that they were written by Robert :-)

**Pull request merged**